src/Components/FilterBursaries.jsx

```diff
@@ -0,0 +1,37 @@
+//NPM packages
+
+//Project files
+import BursaryItem from "./BursaryItem";
+import bursaries from "../Data/bursaries.json";
+import { useFilter } from "state/FilterContext";
+
+export default function FilterBursaries() {
+  //Local state
+  const { academic_category, month } = useFilter();
+
+  const byMonth = bursaries.filter((item) => item.dueDate === `${month}`);
+  const byCategory = bursaries.filter(
+    (item) => item.category === `${academic_category}`
+  );
+
+  if (month === "All" && academic_category === "All") {
+    return bursaries.map((item) => <BursaryItem key={item.id} item={item} />);
+  } else if (month === "All" && academic_category !== "All") {
+    const filteredList = byCategory;
+    return filteredList.map((item) => (
+      <BursaryItem key={item.id} item={item} />
+    ));
```

Comment:
So all these map functions do the same thing, you could get fancy and extract the function for re-use:
```suggestion
    // declare outside the Bursaries component
    function toBursaryItem(item) {
      return (
        <BursaryItem key={item.id} item={item} />
      );
    }

    // usage inside the component
    return filteredList.map(toBursaryItem);
```